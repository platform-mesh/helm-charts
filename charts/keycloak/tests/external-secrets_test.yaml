suite: external-secrets
templates:
  - external-secrets.yaml
tests:
  - it: matches the snapshot
    template: external-secrets.yaml
    set:
      externalSecrets:
        enabled: true
    asserts:
      - matchSnapshot: {}
  - it: matches the snapshot (no external secrets)
    template: external-secrets.yaml
    set:
      externalSecrets:
        enabled: false
    asserts:
      - matchSnapshot: {}
