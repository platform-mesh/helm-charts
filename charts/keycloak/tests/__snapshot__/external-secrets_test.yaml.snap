matches the snapshot:
  1: |
    apiVersion: external-secrets.io/v1
    kind: ExternalSecret
    metadata:
      name: RELEASE-NAME-keycloak-admin
      namespace: openmfp-system
    spec:
      data:
        - remoteRef:
            conversionStrategy: Default
            key: null
            property: password
          secretKey: secret
      refreshInterval: 10m
      secretStoreRef:
        kind: SecretStore
        name: environment-store
      target:
        creationPolicy: Owner
        deletionPolicy: Retain
        name: keycloak-admin
  2: |
    apiVersion: external-secrets.io/v1
    kind: ExternalSecret
    metadata:
      name: RELEASE-NAME-keycloak-postgres
      namespace: openmfp-system
    spec:
      data:
        - remoteRef:
            conversionStrategy: Default
            key: null
            property: password
          secretKey: password
      refreshInterval: 10m
      secretStoreRef:
        kind: SecretStore
        name: environment-store
      target:
        creationPolicy: Owner
        deletionPolicy: Retain
        name: postgres-admin-password
